pom.xml

```diff
@@ -275,11 +275,11 @@
                         <additionalparam>${javadoc.options}</additionalparam>
                         <attach>true</attach>
                         <doclint>none</doclint>
-                        <doctitle>${project.name}</doctitle>
-                        <windowtitle>${project.name}</windowtitle>
+                        <doctitle>${project.name}, ${project.version}</doctitle>
+                        <windowtitle>${project.name}, ${project.version}</windowtitle>
                         <bottom>
 <![CDATA[
-<p align="left">Copyright &#169; 2019-2020 Eclipse Foundation.<br>Use is subject to
+<p align="left">Copyright &#169; 2019,2020 Eclipse Foundation.<br>Use is subject to
```

Comment:
I think the start year of copyright should be 2018 instead of 2019.

Reply:
Updated, thanks.